config/examples/clustermonitoringoperator.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: cmo.example.com/v1
```

Comment:
I need the correct name for this, some ideas?

Reply:
how about `monitoring.coreos.com/v1`

```
$  oc explain Prometheus
GROUP:      monitoring.coreos.com
KIND:       Prometheus
VERSION:    v1
...
```

Reply:
thanks, I don't know if correct group is that or cmo should go in another one